src/main/java/org/elasticsearch/index/mapper/attachment/AttachmentMapper.java

```diff
@@ -451,7 +451,8 @@ public void parse(ParseContext context) throws IOException {
 
             // #18: we could ignore errors when Tika does not parse data
             if (!ignoreErrors) {
-                throw new MapperParsingException("Failed to extract [" + indexedChars + "] characters of text for [" + name + "]", e);
+                throw new MapperParsingException("Failed to extract [" + indexedChars + "] characters of text for [" + name + "] : "
+                        + e.getMessage());
```

Comment:
Add log before throwing the Exception. See https://github.com/elastic/elasticsearch-mapper-attachments/pull/121#issuecomment-87834822